.eslintrc

```diff
@@ -7,6 +7,7 @@
   },
   "rules": {
     "no-var": "error",
-    "no-console": 0
+    "no-console": 0,
+    "multiline-ternary": 0
```

Comment:
I'm wondering if there's some local copy interaction causing your error. The build is running fine when I remove this.